src/lualdap.c

```diff
@@ -143,21 +143,12 @@ static int option_error (lua_State *L, const char *name, const char *type) {
 }
 
 
-/*
-** Get the field called name of the table at position 2.
-*/
-static void strgettable (lua_State *L, int idx, const char *name) {
-	lua_pushstring (L, name);
-	lua_gettable (L, idx);
-}
-
-
 /*
 ** Get the field named name as a string.
 ** The table MUST be at position 2.
 */
 static const char *strtabparam (lua_State *L, int idx, const char *name, char *def) {
-	strgettable (L, idx, name);
+	lua_getfield(L, idx, name);
```

Comment:
I support this. The function is available in lualdap since b592959c432f31ba3151f2c806e4feee6d69baad, so this is good now.